index.js

```diff
@@ -13,7 +13,9 @@ module.exports = function gulpContains(options) {
 		cb(new gutil.PluginError('gulp-contains', error));
 	};
 
-	return through.obj(function (file, enc, cb) {
+	options.highWaterMark = { highWaterMark: options.highWaterMark } || {};
+
+	return through.obj(options.highWaterMark, function (file, enc, cb) {
```

Comment:
Instead of mutating `options.highWaterMark`, could you make a new object called `throughOptions`? E.g.:

```js
var throughOptions = { highWaterMark: options.highWaterMark };

return through.obj(throughOptions, …
```

I find it a bit confusing that before this line, `options.highWaterMark` is a number, and after, it's an object.